device_backends/include/UnifiedBackendTest.h

```diff
@@ -1605,15 +1605,20 @@ namespace ChimeraTK {
 
       std::cout << "... registerName = " << registerName << std::endl;
       auto reg = d.getTwoDRegisterAccessor<UserType>(registerName, 0, 0, {AccessMode::wait_for_new_data});
+      // a helper to the same register to make sure the sending thread has distributed into the queues before
+      // sending the next data
+      auto helper = d.getTwoDRegisterAccessor<UserType>(registerName, 0, 0, {AccessMode::wait_for_new_data});
 
       // Initial value (stays unchecked here)
       reg.read();
+      helper.read();
```

Comment:
I think it would be more clear to not read the helper here (and in 1621) and instead call `helper.readLatest()` before the `x.setRemoteValue()` in 1635 below, when actually starting to need the helper.

Reply:
done